tests/api/test_get_resource.py

```diff
@@ -0,0 +1,641 @@
+                'commit_date': '2017-04-26T11:44:38+00:00',
+                'hash': '8a63adb248ba633e200067e1ad6dc61931727bad',
+                'log_message': 'Related: #12345 - fix xyz'
+            }
+        ],
+        'reverted_by_commits': [
+            {
+                'author_date': '2017-04-27T11:44:38+00:00',
+                'commit_date': '2017-04-27T11:44:38+00:00',
+                'hash': '5663adb248ba633e205067e1ad6dc61931727123',
+                'log_message': 'Revert: #12345'
+            }
+        ],
+        'severity': 'low',
+        'short_description': 'Some description',
+        'status': 'VERIFIED',
+        'target_milestone': 'rc',
+        'votes': 0
+    }),
+    ('distgitcommit', '8a63adb248ba633e200067e1ad6dc61931727bad', {
+        'author': {
+            'email': 'tom.brady@domain.local',
+            'name': None,
+            'username': 'tbrady'
+        },
+        'author_date': '2017-04-26T11:44:38+00:00',
+        'branches': [
+            {
+                'name': 'some_branch_name',
+                'repo_name': 'some_repo_name',
+                'repo_namespace': 'some_repo_namespace'
+            }
+        ],
+        'children': [
+            {
+                'author_date': '2017-04-27T11:44:38+00:00',
+                'commit_date': '2017-04-27T11:44:38+00:00',
+                'hash': '5663adb248ba633e205067e1ad6dc61931727123',
+                'log_message': 'Revert: #12345'
+            }
+        ],
+        'commit_date': '2017-04-26T11:44:38+00:00',
+        'committer': {
+            'email': 'jsmith@domain.local',
+            'name': None,
+            'username': 'jsmith'
+        },
+        'hash': '8a63adb248ba633e200067e1ad6dc61931727bad',
+        'koji_builds': [
+
+        ],
+        'log_message':'Related: #12345 - fix xyz',
+        'parent':{
+            'author_date': '2017-04-27T11:44:38+00:00',
+            'commit_date': '2017-04-27T11:44:38+00:00',
+            'hash': '1263adb248ba633e205067e1ad6dc61931727c2d',
+            'log_message': 'Related: #12345 - fix xz'
+        },
+        'pushes': [
+            {
+                'id': '56789',
+                'push_date': '2017-08-01T15:43:51+00:00',
+                'push_ip': '28.03.19.94'
+            }
+        ],
+        'related_bugs': [
+            {
+                'classification': 'Red Hat',
+                'creation_time': '2017-04-02T19:39:06+00:00',
+                'id': '272895',
+                'modified_time': '2018-02-07T19:30:47+00:00',
+                'priority': 'low',
+                'product_name': 'Satellite',
+                'product_version': '3',
+                'resolution': '',
+                'severity': 'medium',
+                'short_description': 'Some description',
+                'status': 'VERIFIED',
+                'target_milestone': 'rc',
+                'votes': 0
+            },
+            {
```

Comment:
This bug is repeated in resolved bugs. This can be removed  and disconnected in the code.

Reply:
NVM, this is fine for test data.